### DISCUSSION TITLE
Is there a way for flow launcher to remember the previously launched item from a search?

### DISCUSSION THREAD

**Discussion by mx3214** [Flow-Launcher/Flow.Launcher#2325] [Q&A] [answered]
I'm moving over from launchy, and in launchy if you searched for a document eg 'my resume'. And clicked on the pdf of your resume, in the future, when you searched 'my resume', the pdf would be the first option.

However in flow launcher this doesnt seem to work. For example if I type 'my resume' and two options come up, google search for 'my resume' and 'my resume.pdf' and I select the pdf. The next time I type 'my resume' the google search option still comes at the top.

Is there a way to make flow launcher remember previously selected responses? eg make 'my resume.pdf' as the top result

![image](https://github.com/Flow-Launcher/Flow.Launcher/assets/14053275/d1f6443f-9324-4339-9e70-2853d39cb50a)

**Comment by jjw24** [comment, marked as answer]
Hi there and welcome. 

Flow does remember frequently used results and put them on top, this feature however takes a few repetitions of selecting the results. The more you select the higher the selected result will get. 

Alternatively if you would like a particular result to show up at the top always, you can `shift+enter` then select the pin to top option on the result.

A side note, you can also adjust the priority of a plugin's result in the Settings window.

**Comment by manusgeek** [comment]
That would be a great feature. As far as I'm concerned, that's the only thing where launchy is still better than flowlauncher currently. We still have to fully type the whole file name for some items, even if frequently used. Launchy was much better at finding regularly accessed items after typing only the first few letters, a bit like an internet browser search bar.
